packages/blockchain-api/test/mockTokenTxs.ts

```diff
@@ -586,6 +586,41 @@ const mockTokenTxs = {
             transactionHash: '0xfe39014b70746259a1dc4cc99c67acbb986d68f32cdb42e68a2678082a1695dc',
           },
         },
+        // Escrow MTW
+        {
+          node: {
+            blockNumber: 6203333,
+            celoTransfer: {
+              edges: [
+                {
+                  node: {
+                    fromAddressHash: '0x0000000000000000000000000000000000a77327',
+                    toAddressHash: '0xc85d6ccf22499898ea76aa8b7ba89a7ceddbe434',
+                    token: 'cUSD',
+                    value: '20000000000000000000',
+                  },
+                },
+                {
+                  node: {
+                    fromAddressHash: '0xc85d6ccf22499898ea76aa8b7ba89a7ceddbe434',
+                    toAddressHash: '0x566ce6b765f038a98b5753ced0a65fd49aa0a07c',
+                    token: 'cUSD',
+                    value: '20000000000000000000',
+                  },
+                },
+              ],
+            },
+            feeToken: 'cUSD',
```

Comment:
You can't just change it `cUSD` as it would be `cGLD` in Blockscout and cUSD transactions would contain additional transfers (which are not currently present in the `edges` array). If you change it back to `cGLD`, you'll notice that the transaction type is not recognized again, and, hence, is missing from the feed in both test cases.

Also, the code relies on the presence of the `toAccountHash` attribute that is not here and you haven't updated the graphQL query to fetch it from Blockscout.